Customer.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <string>
+
+using namespace std;
+
+class Customer {
+private:
+  string id;
+  string name;
+  string address;
+  string phoneNumber;
+  // rentalList not implemented
+  string rentalList;
+
```

Comment:
Suggestion: 
- An attribute to track current successful return in one session
![image](https://user-images.githubusercontent.com/51154273/103322557-fc9ef300-4a70-11eb-84c3-889c4e135aa9.png)